vendor/primitives/evm-tracing-events/src/runtime.rs

```diff
@@ -225,6 +225,9 @@ pub fn opcodes_string(opcode: Opcode) -> Vec<u8> {
         Opcode(89) => "MSize",
         Opcode(90) => "Gas",
         Opcode(91) => "JumpDest",
+        Opcode(92) => "TLoad",
```

Comment:
Is frontier updated to support this Opcode?

Reply:
This doesn't do any harm though

Reply:
You’re right. Actually, they were introduced recently in https://github.com/moonbeam-foundation/evm/pull/3 by the Moonbeam team in their own EVM fork https://github.com/moonbeam-foundation/frontier/commit/a5483d1f38b2161d8d90f240fdbfa89342b48723.

It might be better to include this in the next uplift to **polkadot-stable2407**. I removed them to avoid future confusion.